MantidQt/SliceViewer/src/QPeaksTableModel.cpp

```diff
@@ -309,14 +310,15 @@ std::vector<int> QPeaksTableModel::defaultHideCols() {
     if (numToCheck > 20) // arbitrary cutoff
       numToCheck = 20;
     const std::string RECT_DET("RectangularDetector");
+    const auto &detectorInfo = m_peaksWS->detectorInfo();
```

Comment:
This code doesn't look like it is doing anything at all; even before the changes.

It just breaks out of the loop if it finds a rectangular detector but I can't see how that affects anything. Am I missing something?